.github/workflows/Tutor_CI.yml

```diff
@@ -0,0 +1,24 @@
+name: Tutor_CI
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        chmod +x ./bootstrap.sh
+        ./bootstrap.sh
```

Comment:
bootstrap.sh script should already have the execute bit on. If not, than it should be taken care of by pushing the boostrap.sh script with the correct permissions.

Reply:
And, what is the value of running bootstrap.sh now? linting doesn't require it.

Reply:
@ifireball  @lioramilbaum 
I'm a bit confused. 
in the beggining I didn't call bootstrap.sh and just had the lines
        python -m pip install --upgrade pip
        pip install flake8
then Barak's review was that I should run bootstrap.sh so I changed it.
so I'm not sure now whether to install through boostrap or locally in this file

Reply:
The first line might be covered by bootstrap.sh but the second one shouldn't be. You should install it outside of bootstrap.sh.
Therefor, I don't see any reason (for now) to run the script. @ifireball thoughts?